### PR TITLE
Add functions `channelAftertouch` & `polyphonicAftertouch` to QMidiOut

### PR DESCRIPTION
**Pull request by paulmasri**

### CODE REVIEW

File: src/QMidiOut.h [hunk 1]
```diff
@@ -26,6 +26,8 @@ class QMidiOut
 	void noteOn(int note, int voice, int velocity = 64);
 	void noteOff(int note, int voice);
 	void pitchWheel(int voice, int value);
+    void channelAftertouch(int voice, int value);
+    void polyphonicAftertouch(int note, int voice, int value);
```
Comment (waddlesplash) [inline src/QMidiOut.h]: Indent by 1 tab, please.

File: src/QMidiOut.cpp [hunk 2]
```diff
@@ -54,6 +54,21 @@ void QMidiOut::pitchWheel(int voice, int value)
 	sendMsg(msg);
 }
 
+void QMidiOut::channelAftertouch(int voice, int value)
+{
+    qint32 msg = 0xD0 + voice;
+    msg |= value << 8;
+    sendMsg(msg);
```
Comment (waddlesplash) [inline src/QMidiOut.cpp]: All of this (and the below) needs to be indented by one tab, and not four spaces.

**Comment by waddlesplash** [comment]
Looks good, aside from the formatting issue.

**Comment by paulmasri** [comment]
👀

File: src/QMidiOut.h (hunk 1 above)
Comment (waddlesplash) [inline src/QMidiOut.h]: Uh, same here?

Comment (paulmasri) [reply src/QMidiOut.h]: D'oh! Now done :-)

**Comment by waddlesplash** [comment]
Thank you!

**Pull request merged**